### PR TITLE
Bump scheduler java runtime to Java 11

### PR DESCRIPTION
**Pull request by takirala**
This PR bumps the jre for schedulers to java 11 and the framework jre is still at java 8.

Also credits to initial work @kaiwalyajoshi did via #3027

### CODE REVIEW

File: tools/universe/package_builder.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 logging.basicConfig(level=logging.DEBUG, format="%(message)s")
 
 _jre_url = "https://downloads.mesosphere.com/java/openjdk-jre-8u212b03-hotspot-linux-x64.tar.gz"
+_scheduler_jre_url = "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.4%2B11/OpenJDK11U-jdk_x64_linux_hotspot_11.0.4_11.tar.gz"
```
Comment (kaiwalyajoshi) [inline tools/universe/package_builder.py]: We want to bring this into our s3 repos and pull from there.

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # See Dockerfile.base for instructions on how to update this base image.
-FROM mesosphere/dcos-commons-base:latest@sha256:0bb843992597e25056548125f979417ce6048b018cc004d8ecab4492732302ab
+FROM mesosphere/dcos-commons-base:latest@sha256:da062e485be6d1b3df081ffcda4d800ede6650bcaed86eb67adc9db169cb2082
```
Comment (takirala) [inline Dockerfile]: This is taken from #3183

File: build.gradle [hunk 3]
```diff
@@ -10,6 +10,8 @@ allprojects {
     apply plugin: 'jacoco'
     apply plugin: 'idea'
     apply plugin: 'maven-publish'
+    // Run `./gradlew htmlDependencyReport` to view the dependency report
```
Comment (kaiwalyajoshi) [inline build.gradle]: (nit) dead comment?

File: test.sh [hunk 4]
```diff
@@ -540,7 +540,14 @@ fi
 ######################### Prepare and run command ##############################
 ################################################################################
 
-docker pull "${docker_image}"
+if [[ "$(docker images -q ${docker_image} 2> /dev/null)" == "" ]]; then
+  echo "Docker image ${docker_image} does not exist locally"
+  # TODO delete this line:
+  docker images
```
Comment (kaiwalyajoshi) [inline test.sh]: (nit) cleanup comments?

File: build.gradle (hunk 3 above)
Comment (takirala) [reply build.gradle]: `htmlDependencyReport` is not intuitive to remember  and i i feel this may be useful in future, keeping this comment.

**Pull request merged**